docs-src/content/audio-input/_index.md

```diff
@@ -264,6 +264,19 @@ stream = client.new_transcribe_stream(action)
 Diatheke::TranscribeStream stream = client.newTranscribeStream(action);
 {{< /tab >}}
 
+{{< tab "Swift/iOS" "swift" >}}
+// Create the transcribe stream. This automatically sends the transcribe
+// action on the stream first.
+let stream = self.newTranscribeStream(action: action) { result in
+	// [Handle transcribe results](#transcribe-results)
```

Comment:
I'm not sure if this works for code snippets. )

Reply:
I don't think it will. Let's just copy the code that is in the transcribe results section here. It will be duplicated, but I think that will be ok.

Reply:
Ok.